website/content/docs/import/mappings.mdx

```diff
@@ -0,0 +1,119 @@
+---
+layout: docs
+page_title: Secrets import mappings
+description: Mappings lets users apply various filtering methods to secrets being imported in to Vault.
+---
+
+# Import mappings
+
+There are multiple types of mapping blocks that can be defined in the import plan. Each type enables a
+different mechanism for filtering out secrets from an external system during the execution of an import.
+
+## Argument reference
```

Comment:
We have two options here:
1. Keep the "Argument reference" section but rename it to something like "Common mapping fields", document the common fields in a partial, then include the partial here and in the import/index.mdx file (per earlier suggested edits)
2. Remove this section and focus just on the filter-specific fields since the common fields are already documented in the general import doc

Reply:
I don't think I intended to have it in both places, and your suggestion #2 makes the most sense to me because of that. 👍